### DISCUSSION TITLE
hawtio war that includes hawtio-log plugin

### DISCUSSION THREAD

**Discussion by amanda45762** [hawtio/hawtio#2909] [Q&A] [answered]
Hi, can anyone help me with creating or locating a hawtio war that includes the hawtio-log plugin? I have hawtio-war-2.17.0.war but when i unzip it and go to WEB-INF/lib I don't see hawtio-log, even though the hawtio website says a list of plugins ( that includes hawtio-log) should be included.

**Comment by tadayosi** [comment]
You can use `hawtio-default.war` that includes hawtio-log.

**Reply by amanda45762** [reply]
can u please provide a link? thank you

**Reply by tadayosi** [reply, marked as answer]
https://repo1.maven.org/maven2/io/hawt/hawtio-default/2.17.6/hawtio-default-2.17.6.war

**Reply by amanda45762** [reply]
how can I see event logs? Ultimately trying to track which users execute which jmx commands via hawtio. I'm running hawtio in tomcat

**Reply by tadayosi** [reply]
Currently it only supports logback and you also need to include https://repo1.maven.org/maven2/io/hawt/hawtio-log-logback/2.17.6/hawtio-log-logback-2.17.6.jar in the lib to make it work. If you use different logging frameworks it's unfortunately not available for now.
